### DISCUSSION TITLE
PROJ Runtime Error

### DISCUSSION THREAD

**Discussion by Warren-Whitfield** [pelicanmapping/osgearth#2536] [Support Q&A]
Any idea why I get this runtime error? It doesn't seem to affect the operation, but it is generated every time the map is zoomed or moved. I've made sure proj is installed and as far as I'm aware everything seems to be fine.

Warning 1: PROJ: proj_create_from_database: Cannot find proj.db
ERROR 1: PROJ: proj_identify: Cannot find proj.db

**Comment by gwaldron** [comment]
You can set the environment variable `PROJ_LIB` to point at the folder containing proj.db.
This is typically in a `share/proj` folder wherever GDAL was installed.

Why you are getting the message, I don't know
